src/ansys/simai/core/utils/configuration.py

```diff
@@ -36,6 +45,13 @@ def prompt_for_input_factory(*args, **kwargs):
     return lambda: prompt_for_input(*args, **kwargs)
```

Comment:
I can't select it but the `prompt_for_input_factory` is probably not used anymore and could be removed ?

Reply:
yup, it'll be removed